pkg/ast/spl/spl.peg

```diff
@@ -523,6 +523,81 @@ TextExpr <- (opName:("lower") "(" stringExpr:StringExpr ")") {
     }
     return node, nil
 }
+/ (opName:("substr") "(" stringExpr:StringExpr "," startIndex:NumericExpr lengthParam:(", " NumericExpr)? ")") {
```

Comment:
Make sure you allow a space after the comma (you did this for the second comma but not the first)

Reply:
Okay will do that